src/main/java/com/ldtteam/blockout/controls/ImageRepeatable.java

```diff
@@ -0,0 +1,155 @@
+package com.ldtteam.blockout.controls;
+
+import com.ldtteam.blockout.Pane;
+import com.ldtteam.blockout.PaneParams;
+import com.mojang.blaze3d.matrix.MatrixStack;
+import com.mojang.blaze3d.systems.RenderSystem;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.Tuple;
+import org.lwjgl.opengl.GL11;
+
+/**
+ * Image element with repeatable middle part.
+ */
+public class ImageRepeatable extends Pane
```

Comment:
Surely we can extend the Image functionality, if not integrate the two?

Reply:
image is scalable, this is not (or not supposed to be), so integrating can be difficult, but for sure something can be abstracted
same goes towards ButtonImage - there is a problem of having the image logic three times duplicated
leaving the decision up to you

Reply:
Hmm, I think my PR will solve this anyway, it will allow for property grouping of sorts. I'll let it explain itself once I push it.

Reply:
Marking as resolved for now